crux-mir/src/Mir/Language.hs

```diff
@@ -247,7 +247,7 @@ runTestsWithExtraOverrides initS bindExtra (cruxOpts, mirOpts) = do
                 ifs -> error $ "expected exactly 1 input file, but got " ++ show (length ifs) ++ " files"
 
     -- Load the MIR collection
-    col <- generateMIR cruxOpts filename False
+    col <- generateMIR cruxOpts (mirJsonArgs mirOpts) filename False
```

Comment:
A bit of a personal stylistic preference, but I would rather `generateMIR` take a `MIROptions` value instead of a `[String]`, as it is more obvious what the purpose of the former type is than the latter.